version.properties

```diff
@@ -1 +1 @@
-VERSION_TAG=3.40.1
+VERSION_TAG=3.41.0
```

Comment:
Does this version need to strictly match the version of the Simulation API Specification it's targeting? If so, I need to change this to 3.44.0.